src/components/tag/AposTag.vue

```diff
@@ -1,38 +1,46 @@
 <template>
   <div :class="`apos-tag ${modifier}`" :disabled="disabled">
```

Comment:
The `disabled` attribute doesn't do anything anymore.

Reply:
Can't tell if it ever did. Maybe this used to be a button... Should this be a button? I think that might make sense. It does a thing when you click it. The alternative would be to make only the `x` a button.

Reply:
So then we should be emitting an event when it's clicked as well.

Reply:
going to make the x a button, clicking the whole thing to remove feels bad